cspell/.cspell.json

```diff
@@ -297,6 +297,7 @@
         "FASTCRC",
         "favicon",
         "fclose",
+        "feedback",
```

Comment:
```suggestion
```
`feedback` has been already included in the default en_US dictionary.

Reply:
Could you tell me the place of the en_US dictionary ? I also confirm it.

Reply:
Is this?
https://raw.githubusercontent.com/streetsidesoftware/cspell-dicts/main/dictionaries/cpp/cpp.txt
I can find `feedback`.
So I delete it.